extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -895,6 +904,28 @@ export class AmpAutocomplete extends AMP.BaseElement {
     }
   }
 
+  /**
+   * If a fallback element is provided, displays it instead of suggestions.
+   * Otherwise, throws given error. Must be called in a mutate context.
+   * @param {*=} error
+   * @throws {!Error} If fallback element is not present.
+   * @private
+   */
+  displayFallback_(error) {
+    if (this.fallbackDisplayed_) {
+      return;
+    }
+    this.clearAllItems_();
+    const fallback = this.getFallback();
+    if (fallback) {
+      this.fallbackDisplayed_ = true;
+      this.toggleFallback(true);
+      user().warn(TAG, 'Triggered fallback due to error:', error);
```

Comment:
It should be OK to do without these warnings. I don't see other components emitting warnings when showing the fallback, so you can follow the same pattern here